### PR TITLE
Posttraining documentation refine

### PR DESCRIPTION
**Pull request by hsuan-lun-chiang**
# Description

Improves post-training documentation for multimodal models and corrects file paths in the example commands.

# Tests

Manually verified the documentation by running the commands for Checkpoint Conversion, Multimodal Decoding, and Supervised Fine-Tuning to ensure correctness.

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code. For an optional AI review, add the `gemini-review` label.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run end-to-end tests tests and provided workload links above if applicable.
- [X] I have made or will make corresponding changes to the doc if needed, including adding new documentation pages to the relevant Table of Contents (toctree directive) as explained in [our documentation](https://maxtext.readthedocs.io/en/latest/development.html#adding-new-documentation-files).

### CODE REVIEW

File: docs/tutorials/posttraining/multimodal.md [hunk 1]
```diff
@@ -25,7 +25,14 @@ Multimodal Large Language Models (LLMs) extend traditional text-only models by i
 
 ## Checkpoint Conversion
 
-Recently we have onboarded a new centralized tool for bidirectional checkpoint conversion between MaxText and HuggingFace ([README](https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/utils/ckpt_conversion/README.md)). This tool is used for the Gemma3 model family. Use this command to convert an unscanned checkpoint from HuggingFace to MaxText, and save it to `MAXTEXT_CKPT_GCS_PATH`:
+Recently we have onboarded a new centralized tool for bidirectional checkpoint conversion between MaxText and HuggingFace ([README](https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/utils/ckpt_conversion/README.md)). This tool is used for the Gemma3 model family. 
+
+Install pytorch:
+```
+pip install torch
```
Comment (SurbhiJainUSC) [inline docs/tutorials/posttraining/multimodal.md]: `python3 -m pip install torch --index-url https://download.pytorch.org/whl/cpu`

File: docs/tutorials/posttraining/multimodal.md [hunk 2]
```diff
@@ -98,6 +105,8 @@ python -m MaxText.decode \
     MaxText/configs/base.yml \
     model_name=gemma3-4b \
     ... \
+    max_prefill_predict_length=predict_length  # Adjust to fit image tokens + text prompt \
```
Comment (SurbhiJainUSC) [inline docs/tutorials/posttraining/multimodal.md]: Where do we initialize `predict_length` and `target_length`?

File: docs/tutorials/posttraining/multimodal.md [hunk 3]
```diff
@@ -134,6 +143,30 @@ python -m MaxText.sft_trainer \
     dataset_type=hf profiler=xplane
 ```
 
+Also, we can use the converted unscanned checkpoint from previous `Checkpoint Conversion` section.
+
+```
+python -m MaxText.sft_trainer \
+    $MAXTEXT_REPO_ROOT/configs/sft-vision-chartqa.yml \
```
Comment (SurbhiJainUSC) [inline docs/tutorials/posttraining/multimodal.md]: `src/MaxText` instead of `$MAXTEXT_REPO_ROOT`

File: docs/tutorials/posttraining/multimodal.md [hunk 4]
```diff
@@ -134,6 +143,30 @@ python -m MaxText.sft_trainer \
     dataset_type=hf profiler=xplane
 ```
 
+Also, we can use the converted unscanned checkpoint from previous `Checkpoint Conversion` section.
```
Comment (SurbhiJainUSC) [inline docs/tutorials/posttraining/multimodal.md]: As a reader, I got confused with this new SFT command. Instead of this, should we modify `$UNSCANNED_CKPT_PATH` in the above command and explicitly mention that this variable can be either set to an already available MaxText ckpt or to the one we just converted in the previous step.

File: docs/tutorials/posttraining/multimodal.md [hunk 5]
```diff
@@ -25,7 +25,14 @@ Multimodal Large Language Models (LLMs) extend traditional text-only models by i
 
 ## Checkpoint Conversion
 
-Recently we have onboarded a new centralized tool for bidirectional checkpoint conversion between MaxText and HuggingFace ([README](https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/utils/ckpt_conversion/README.md)). This tool is used for the Gemma3 model family. Use this command to convert an unscanned checkpoint from HuggingFace to MaxText, and save it to `MAXTEXT_CKPT_GCS_PATH`:
+Recently we have onboarded a new centralized tool for bidirectional checkpoint conversion between MaxText and HuggingFace ([README](https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/utils/ckpt_conversion/README.md)). This tool is used for the Gemma3 model family.
```
Comment (SurbhiJainUSC) [inline docs/tutorials/posttraining/multimodal.md]: `This tool is used for the Gemma3 model family. ` is confusing. Checkpoint conversion tool is available for other model family as well.

File: docs/tutorials/posttraining/multimodal.md (hunk 2 above)
Comment (hsuan-lun-chiang) [reply docs/tutorials/posttraining/multimodal.md]: Thanks! I updated the example to export those variables before the command.

File: docs/tutorials/posttraining/multimodal.md (hunk 1 above)
Comment (hsuan-lun-chiang) [reply docs/tutorials/posttraining/multimodal.md]: Done. Thanks!

File: docs/tutorials/posttraining/multimodal.md (hunk 3 above)
Comment (hsuan-lun-chiang) [reply docs/tutorials/posttraining/multimodal.md]: Done, thanks!

File: docs/tutorials/posttraining/multimodal.md (hunk 5 above)
Comment (hsuan-lun-chiang) [reply docs/tutorials/posttraining/multimodal.md]: I have removed the text. Thanks.

File: docs/tutorials/posttraining/multimodal.md (hunk 4 above)
Comment (hsuan-lun-chiang) [reply docs/tutorials/posttraining/multimodal.md]: Thanks! I've updated the example to export `$UNSCANNED_CKPT_PATH` and added a comment explaining it.

File: docs/tutorials/posttraining/multimodal.md (hunk 3 above)
Comment (A9isha) [reply docs/tutorials/posttraining/multimodal.md]: Curious: are we moving away from MAXTEXT_REPO_ROOT?
